recipe/build.sh

```diff
@@ -1,3 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+
+if [[ $(uname -m) == "ppc64le" ]]; then
+      sed -i.bak 's,march=native,mcpu=powerpc64le,' GKlibSystem.cmake
+fi
+
```

Comment:
Could we set this via the `GKlib_COPTIONS` env var rather than patching?

Reply:
I think that won't work, since `gcc` doesn't support `-march=native` at all on `ppc64le`, and adding another flag won't change that behavior. We therefore need to remove or replace this option, as done here.

Also, it seems that the build system does not honor `GKlib_COPTIONS` set in the environment, and always overrides it.